sdks/urbackend-python/test_sdk.py

```
@@ -0,0 +1,380 @@
+    try:
+        UrBackendClient(api_key=API_KEY, base=BASE).auth.login(
+            "fake@fake.com", "wrong"
+        )
+        print("Invalid login didn't raise!")
+    except AuthError as e:
+        print(f"Invalid login → AuthError (status={e.status_code}): {e.message}")
+    except ValidationError as e:
+        print(f"Invalid login → ValidationError (status={e.status_code}): {e.message}")
+        print("   Note: Backend returns 400 for bad login, which maps to ValidationError.")
+        print("   This is acceptable — callers should catch UrBackendError base class.")
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)